bolt/include/bolt/Core/Relocation.h

```diff
@@ -50,72 +55,79 @@ struct Relocation {
   /// Used to validate relocation correctness.
   uint64_t Value;
 
+  /// Relocations added by optimizations can be optional.
+  bool Optional = false;
```

Comment:
Move the new flag next to Type, such that the size of `struct Relocation` doesn't change.

Reply:
Resolving this comment as this is now done by:
- #131638